core/src/main/java/cucumber/runner/TimeServiceEventBus.java

```diff
@@ -0,0 +1,14 @@
+package cucumber.runner;
+
+public class TimeServiceEventBus extends AbstractEventBus {
```

Comment:
Replacement for the default event bus. This is actually the only event bus that can be created without needing another event bus.